powersimdata/input/grid.py

```diff
@@ -19,12 +19,11 @@ class Grid:
 
     """Grid
 
-    :param str/iterable interconnect: geographical region covered. Either *'USA'*, one of
-        the three interconnections, i.e., *'Eastern'*, *'Western'* or *'Texas'* or a
+    :param str/iterable interconnect: geographical region covered. Either *'USA'*, one
+        of the three interconnections, i.e., *'Eastern'*, *'Western'* or *'Texas'* or a
         combination of two interconnections.
-    :param str source: model used to build the network. Can be one of the
-        supported models ("usa_tamu"), or a .mat file that represents a
-        grid.
+    :param str source: model used to build the network. Can be one of the supported
+        models ("usa_tamu"), or a .mat file that represents a grid.
     :param str engine: engine used to run scenario, if using ScenarioGrid.
     :raises TypeError: if source and engine are not both strings.
     :raises ValueError: if source or engine does not exist.
```

Comment:
I think this docstring needs updating for the content of the `interconnect` and `source` descriptions. I neglected to do it when I initially enabled the `hifld` grid, sorry.

Reply:
Done. I removed the parentheses since the mention of supported models is explicit enough

Reply:
What about `interconnect` though? We can pass `"Texas"` with `"usa_tamu"` or `"ERCOT"` with `"hifld"` but we can't mix them. Not sure whether we want to go into a lot of detail here or whether we want to point to somewhere else that has allowable combinations like `powersimdata/network/constants/model.model2interconnect`.

Reply:
Indeed. I changed it and it should be grid model independent now (this will help for Europe)